pkg/helper/labels.go

```diff
@@ -13,18 +11,15 @@ const (
 	InfrastructureType ComponentType = "infrastructure"
 )
 
-func MeteringLabels(componentName, componentVersion string, componentType ComponentType) map[string]string {
-	labels := map[string]string{
-		"com.redhat.product-name":      "3scale",
-		"com.redhat.component-name":    componentName,
-		"com.redhat.product-version":   product.ThreescaleRelease,
-		"com.redhat.component-type":    string(componentType),
-		"com.redhat.component-version": componentVersion,
-	}
-
-	if len(validation.IsValidLabelValue(componentVersion)) > 0 {
-		labels["com.redhat.component-version"] = ""
+func MeteringLabels(componentName string, componentType ComponentType) map[string]string {
+	return map[string]string{
+		"com.company":   "Red_Hat",
+		"rht.prod_name": "Red_Hat_Integration",
+		// How to make this version match RHI version? Env var and data from env var?
+		"rht.prod_ver":  "2021.Q4",
```

Comment:
@miguelsorianod  this `2021.Q4` is specific for the 2.12 release. 

Maybe should I fill with `master` and change in the `3scale-2.12-` branch?

Reply:
Seems reasonable to me. We do the same with other labels

Reply:
changed to master